run_uh.py

```diff
@@ -294,53 +295,55 @@ def import_fife(paths):
 		return False
 	return True
 
-def find_fife():
-	# Use the path the user provided.
+def get_fife_paths():
+	"""Returns all possible paths to search for the fife module. New paths to be added as needed."""
+	# Use the path the user provided:
 	from horizons.util.cmdlineoptions import get_option_parser
 	options = get_option_parser().parse_args()[0]
+	# All paths collected in a list:
+	paths = []
+	# Check if path was provided by user:
 	if options.fife_path:
 		fife_path = os.path.abspath(options.fife_path)
+		paths.append(fife_path)
 		# Support giving the path to FIFE_ROOT/engine/python/fife/__init__.pyc etc.
+
```

Comment:
That newline is confusing, the comment is for the code below, so it should be removed.